Fitbit.Portable/FitbitClient.cs

```diff
@@ -1030,6 +1030,11 @@ public Task<TimeSeriesDataListInt> GetTimeSeriesIntAsync(TimeSeriesResourceType
         /// <returns></returns>
         public async Task<HeartActivitiesTimeSeries> GetHeartRateTimeSeries(DateTime date, DateRangePeriod dateRangePeriod, string userId = "-")
         {
+            if (string.IsNullOrWhiteSpace(userId))
+            {
+                userId = "-";
+            }
+
             string path = $"1.1/user/{userId}/activities/heart/date/{date:yyyy-MM-dd}/{dateRangePeriod.GetStringValue()}.json";

```

Comment:
This change looks like it was needed because the path isn't being built up using the FitbitClientHelperExtensions.ToFullUrl which does this check for the userId. Example below from GetActivitiesStatsAsync ...

```
        public async Task<ActivitiesStats> GetActivitiesStatsAsync(string encodedUserId = null)
        {
            string apiCall = FitbitClientHelperExtensions.ToFullUrl("/1/user/{0}/activities.json", encodedUserId);
```

Reply:
Makes sense to me Adam! Dwayne and I actually had a brief conversation along the lines of "there really should be a helper..." but this was a quick in & out job for me yesterday so I missed it. I'll fix the PR today to leverage the helper instead, thanks!

Reply:
Better documentation may help contributors to understand what needs to be passed in.
```/// <summary>
        /// Converts the REST api resource into the fully qualified url
        /// </summary>
        /// <param name="apiCall">
             ///"Format" string with one "Format Item" as a placeholder for "UserId"
             ///</param>
        /// <param name="encodedUserId"></param>
        /// <param name="args"></param>
        /// <returns>Fully qualified url</returns>```